ui-design-lib/src/main/java/co/electriccoin/zcash/ui/design/component/TextField.kt

```diff
@@ -31,40 +32,53 @@ fun FormTextField(
     value: String,
     onValueChange: (String) -> Unit,
     modifier: Modifier = Modifier,
+    error: String? = null,
+    enabled: Boolean = true,
     textStyle: TextStyle = ZcashTheme.extendedTypography.textFieldValue,
-    label: @Composable (() -> Unit)? = null,
-    leadingIcon: @Composable (() -> Unit)? = null,
-    trailingIcon: @Composable (() -> Unit)? = null,
+    placeholder:
+        @Composable()
+        (() -> Unit)? = null,
+    leadingIcon:
+        @Composable()
+        (() -> Unit)? = null,
```

Comment:
What leadingIcon does? Seems like it's something from the previous design we had months or even years ago. Do we want to keep it or clean up the code?

Reply:
The TextField is a reusable component accessible from our `ui-design-lib` module, which makes it an API with a different use case across the app's UI. Having the leading icon parameter defined as nullable is fine even if not used. The `trailingIcon` parameter is used on the Send.Form for the recipient address TextField, so logically, the API could also provide the `loadingIcon.`